crates/driver/src/domain/mempools.rs

```diff
@@ -40,7 +45,13 @@ impl Mempools {
                 ?auction_id,
             ))
             .boxed()
-        })));
+        }));
+
+        let solver = solver.clone();
+        tokio::task::spawn(async move {
+            let result = futures.await.map(|(result, _)| result);
+            notify::mempools_executed(&solver, auction_id, &result);
```

Comment:
nit: just `executed` has as much information and is shorter I find

Reply:
renamed.